src/mawpy/steps/incremental_clustering.py

```diff
@@ -215,7 +211,7 @@ def _k_means_cluster_lloyd(cluster_list):
     return cluster_list_new
 
 
-def _get_clusters(locations_for_clustering, spat_constr):
+def _get_clusters(locations_for_clustering: list[tuple[str, str]], spat_constr: float) -> list[Cluster]:
```

Comment:
Confirming we don't need to support python <3.9? I believe using the lower case hints is supported on >=3.9.

Reply:
The package has been designed keeping this in mind:
https://scientific-python.org/specs/spec-0000/

Anyhow, we can confirm this with ThinkLab team as well.

Reply:
Ok, I see the pyproject.toml has this:
```
requires-python = ">=3.11"
```